build/ci/evergreen.yml

```diff
@@ -466,6 +466,24 @@ tasks:
           MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
           MCLI_SERVICE: cloud
           E2E_TAGS: atlas,clusters
+  # LDAP Configuration depends on a cluster running
```

Comment:
[q] do they?

Reply:
yes, they do.
<img width="1282" alt="Screenshot 2020-09-14 at 10 57 09" src="https://user-images.githubusercontent.com/5663078/93065487-10b51500-f679-11ea-88e2-91f988c21c72.png">

Reply:
weird but I'll take it 👍